staff/daniel-toledo/cv/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Daniel Toledo</title>
+</head>
+<body>
+    <header>
+        <h1>Curriculum Vitae Daniel Toledo</h1>
+        <h3>Renewable Energy Engineer and Future Javascript Programmer</h3>
+        <p>30th April 1992</p>
+        <p>Santa Catalina 18 1r, 08014 Barcelona, Spain</p>
+        <p><b>Tel :</b> +34 692461556</p>
+        <a href="mailto:daniel.toledomonfort@gmail.com">daniel.toledomonfort@gmail.com </a>
+        <dt><a href="https://www.linkedin.com/in/danieltoledomonfort/">Linkedin</a></dt>
+    </header>
+    <br>
+    <section>
+        <hr>
+        <h2><u>Experience</u></h2>
+        <br>
+        <ul>
+            <li>
+                <p><em>2017 Lisbon-Portugal</em></p>
+                <h3><b>Design Optimization of Mooring System for the WindFloat Technology</b></h3>
```

Comment:
The element is used to define bold (text). This is to style it.
According to the HTML 5 specification, the tag should be used as a LAST resort when no other tag is more appropriate. The HTML 5 specification states that headings should be denoted with the

The element is used to define bold (text). This is to style it.
According to the HTML 5 specification, the tag should be used as a LAST resort when no other tag is more appropriate. The HTML 5 specification states that headings should be denoted with the to
tags, emphasized text should be denoted with the tag, important text should be denoted with the tag, and marked/highlighted text should use the tag. https://www.w3schools.com/tags/tag_b.asp